addons/m252_mortar/model.cfg

```diff
@@ -1,40 +1,137 @@
         {
-
+            "base_plate","",
+            "mortar_tube","base_plate",
+            "support_top","mortar_tube",
+            "support_base","support_top",
+            "deflection_handwheel","support_top",
+            "deflection_grip","deflection_handwheel",
+            "elevation_handwheel","support_base",
+            "elevation_grip","elevation_handwheel"
         };
     };
 };
 
-class CfgModels 
+class CfgModels
 {
-    class Default 
+    class Default
     {
         sectionsInherit = "";
         sections[] = {};
         skeletonName = "";
     };
-    
-    class m252_mortar: Default 
+
+    class m252_mortar: Default
     {
         skeletonName = "M252_Mortar_Skeleton";
         sectionsInherit = "Default";
-        sections[] = {"camo1","camo2","camo3"};
-        
+        sections[] = {"camo1"};
+
         class Animations
         {
-        
+            class main_gun
+            {
+                type = "rotation";
+                source = "main_gun";
+                selection = "main_gun";
+                minValue = 0;
+                maxValue = 1;
+                angle0 = 0;
+                angle1 = 1;
+                axis = "axis_gun";
+            };
+            class main_turret
+            {
+                type = "rotation";
+                source = "main_turret";
+                selection = "main_gun";
+                minValue = -1;
+                maxValue = 1;
+                angle0 = -1;
+                angle1 = 1;
+                axis = "axis_turret";
+            };
+    		class elev_tube1
+            {
+    			type = "rotation";
+    			source = "elevation";
+    			selection = "mortar_tube";
+    			axis = "axis_gun";
+    			minValue = 4.5;
+    			maxValue = 7.0;
+    			angle0 = "rad -10";
+    			angle1 = "rad 15";
+    		};
+    		class elev_bipod_rot1: elev_tube1
+            {
+    			selection = "support_top";
+    			axis = "deflection_axis";
+    			angle0 = "rad -13";
+    			angle1 = "rad 9";
+    		};
+    		class elev_bipod_ext1: elev_tube1
+            {
+    			type = "translation";
+    			selection = "support_base";
+    			axis = "extension_axis";
+    			offset0 = -0.55;
+    			offset1 = 1;
+    		};
+    		class elev_tube2: elev_tube1
+            {
+    			minValue = 7.0;
+    			maxValue = 8.5;
+    			angle0 = "rad 0";
+    			angle1 = "rad 15";
+    		};
+    		class elev_bipod_rot2: elev_tube2
+            {
+                selection = "support_top";
+    			axis = "deflection_axis";
+    			angle0 = "rad 0";
+    			angle1 = "rad 26.5";
+    		};
+    		// WIP for now; this stuff is really difficult
+            class deflection_tube
+            {
+    			type = "rotation";
+    			source = "deflection";
+    			selection = "mortar_tube";
+    			axis = "axis_turret";
+    			minValue = -1;
+    			maxValue = 1;
+    			angle0 = "rad 5.625";
+    			angle1 = "rad -5.625";
+    		};
+    		class deflection_bipod_rot: deflection_tube
```

Comment:
Looks like some tabs slipped through. There's probably a convert tabs > spaces button you can hit in your editor to fix this.